packages/dht/src/connection/ConnectionManager.ts

```diff
@@ -175,9 +178,11 @@ export class ConnectionManager extends EventEmitter<Events> implements ITranspor
         }
         this.state = ConnectionManagerState.RUNNING
         logger.trace(`Starting ConnectionManager...`)
+        const autocertifierRpcCommunicator = new ListeningRpcCommunicator(AUTOCERTIFIER_SERVICE_ID, this)
```

Comment:
Would it be possible to keep `autocertifier` out from `ConnectionManager` (and `ConnectivityChecker`) as the functionalities are quite separate (forming connection / checking connectivity vs. obtaining a self-signed certificate)